app/controllers/concerns/tasklist_ab_testable.rb

```diff
@@ -0,0 +1,72 @@
+module TasklistABTestable
+  TASKLIST_DIMENSION = 66
+
+  TASKLIST_PRIMARY_PAGES = %w(
+    /legal-obligations-drivers-riders
+    /driving-lessons-learning-to-drive/
+    /government/publications/car-show-me-tell-me-vehicle-safety-questions
+    /theory-test/revision-and-practice
+    /theory-test/what-to-take
+    /pass-plus
+  ).freeze
+
+  TASKLIST_SECONDARY_PAGES = %w(
+    /driving-eyesight-rules
+    /complain-about-a-driving-instructor
+    /driving-test-cost
+    /government/publications/driving-instructor-grades-explained
+    /guidance/rules-for-observing-driving-tests
+    /speed-limits
+    /seat-belts-law
+    /vehicle-insurance
+    /report-an-illegal-driving-instructor
+    /government/publications/know-your-traffic-signs
```

Comment:
FYI: #515 is another test that will run on this page. @alecgibson

Reply:
Hmm I'll check that this is okay with Ivan.